src/OrchardCore.Modules/OrchardCore.Liquid/package.json

```diff
@@ -2,6 +2,6 @@
   "name": "orchardcore.liquid",
   "version": "1.0.0",
   "dependencies": {
-    "monaco-editor": "0.35.0"
+    "monaco-editor": "0.37.1"
```

Comment:
I get the same error that I mentioned under https://github.com/OrchardCMS/OrchardCore/pull/13322#discussion_r1131004817

Reply:
@MichaelPetrinolis Do you know where there error come from?
`jsonWorker.js:34 Not allowed to load local resource: file:///OrchardCore.Resources/Scripts/monaco/IStandaloneEditorConstructionOptions.json`